packages/components/slider/src/slider.tsx

```diff
@@ -107,10 +107,15 @@ export type UseSliderProps = FormControlOptions & {
    * Function called whenever the slider value changes.
    */
   onChange?: (value: number) => void
+  /**
+   * aria-valuetext is used to provide a user-friendly text representation of the current value of the slider.
+   */
+  ["aria-valuetext"]?: string
 }
 
 export const useSlider = ({
   focusThumbOnChange = true,
+  "aria-valuetext": ariaValueText,
```

Comment:
Instead of inserting it on this line, insert it under the line `"aria-readonly": ariaReadonly`.